dspace-server-webapp/src/test/java/org/dspace/app/rest/ProcessRestRepositoryIT.java

```diff
@@ -322,6 +324,447 @@ public void getProcessFilesTypesRandomProcessId() throws Exception {
+                            ProcessMatcher.matchProcess(newProcess2.getName(),
+                                                        String.valueOf(eperson.getID().toString()),
+                                                        newProcess2.getID(), parameters, ProcessStatus.SCHEDULED),
+                            ProcessMatcher.matchProcess(newProcess3.getName(),
+                                                        String.valueOf(eperson.getID().toString()),
+                                                        newProcess3.getID(), parameters, ProcessStatus.SCHEDULED)
+                        )))
+                        .andExpect(jsonPath("$.page", is(
+                            PageMatcher.pageEntryWithTotalPagesAndElements(0, 20, 1, 3))));
+    }
+
+    @Test
+    public void searchProcessTestByUserSortedOnMultipleBadRequest() throws Exception {
+        Process newProcess1 = ProcessBuilder.createProcess(context, eperson, "mock-script", parameters)
+                                            .withStartAndEndTime("10/01/1990", "20/01/1990").build();
+        Process newProcess2 = ProcessBuilder.createProcess(context, eperson, "mock-script", parameters)
+                                            .withStartAndEndTime("11/01/1990", "19/01/1990").build();
+        Process newProcess3 = ProcessBuilder.createProcess(context, eperson, "mock-script", parameters)
+                                            .withStartAndEndTime("12/01/1990", "18/01/1990").build();
+
+        String token = getAuthToken(admin.getEmail(), password);
+
+        getClient(token).perform(get("/api/system/processes/search/byProperty")
+                                     .param("userId", eperson.getID().toString())
+                                     .param("sort", "endTime,desc")
+                                     .param("sort", "startTime,desc"))
+                        .andExpect(status().isBadRequest());
+    }
+
+    @Test
+    public void searchProcessTestByUserSortedOnDefault() throws Exception {
+        Process newProcess1 = ProcessBuilder.createProcess(context, eperson, "mock-script", parameters)
+                                            .withStartAndEndTime("10/01/1990", "20/01/1990").build();
+        Process newProcess2 = ProcessBuilder.createProcess(context, eperson, "mock-script", parameters)
+                                            .withStartAndEndTime("11/01/1990", "19/01/1990").build();
+        Process newProcess3 = ProcessBuilder.createProcess(context, eperson, "mock-script", parameters)
+                                            .withStartAndEndTime("12/01/1990", "18/01/1990").build();
+
+        String token = getAuthToken(admin.getEmail(), password);
+
+        getClient(token).perform(get("/api/system/processes/search/byProperty")
+                                     .param("userId", eperson.getID().toString())
+                                     .param("sort", "startTime,desc"))
```

Comment:
With the name `searchProcessTestByUserSortedOnDefault`, I did expect no sort parameters. Did I misinterpret the goal of this method?